jmclient/jmclient/taker.py

```diff
@@ -744,16 +744,16 @@ def make_commitment(self):
         This will allow future upgrades to provide different style commitments
         by subclassing Taker and changing the commit_type_byte; existing makers
         will simply not accept this new type of commitment.
-        In case of success, return the commitment and its opening.
-        In case of failure returns (None, None) and constructs a detailed
-        log for the user to read and discern the reason.
+        In case of success, return the (commitment, commitment opening).
+        In case of failure returns (None, None, err )where 'err' is a detailed
```

Comment:
```suggestion
        In case of failure returns (None, None, err) where 'err' is a detailed
```

Reply:
Fixed in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/969/commits/93da24bf44338c420ef0fc0c37498836a21dc1b5